doc/examples/segmentation/plot_chan_vese.py

```diff
@@ -3,34 +3,21 @@
 Chan-Vese Segmentation
 ======================
 
-The Chan-Vese Algorithm is an algorithm designed to segment objects
-without clearly defined boundaries. This algorithm is based on level
-sets and that are evolved iteratively to minimize an energy, which is
-defined by weighted values corresponding to the sum of differences in
-intensity from the average value outside the segmented region, the sum
-of differences from the average value inside the segmented region, and
-a term which is dependent on the length of the boundary of the
-segmented region.
+The Chan-Vese Algorithm is designed to segment objects without clearly
+defined boundaries. This algorithm is based on level sets that are
+evolved iteratively to minimize an energy, which is defined by
+weighted values corresponding to the sum of differences inintensity
+from the average value outside the segmented region, the sum of
+differences from the average value inside the segmented region, and a
+term which is dependent on the length of the boundary of the segmented
+region.
 
 This algorithm was first proposed by Tony Chan and Luminita Vese, in
-an publicaion entitled "An Active Countour Model Without Edges" [1]_.
+a publicaion entitled "An Active Countour Model Without Edges" [1]_.
 
-This implementation of the algorithm is somewhat simplified.
-The arguments are as follows:
-
-lambda1 : weight for the difference from the average 'inside' the
-          segmented region.
-lambda2 : weight for the difference from the average 'outside' the
-          segmented region.
-mu : weight for the term that is dependent on the length of the
-     boundary of the segmented region
-tol : Convergence tolerance. This defines the stop condition for the
-      Algorithm. A lower value means more iterations and greater
-      accuracy.
-maxiter : maximum number of iterations before the algorithm stops. This
-          is used to avoid endless loops if the solution is divergent.
-dt : a multiplication factor used to speed up the algorithm. Increasing
-     it may cause convergence problems.
+This implementation of the algorithm is somewhat simplified in the
+sense that the area factor 'nu' described in the original paper, and
```

Comment:
@GGoussar it's clear for the reason, but the wording is (IMO) not correct.
This one below is much clearer: "Active contour model by evolving a level set. Supports 2D grayscale images only, and does not implement the area term described in the original article."